gui/slick/views/layouts/main.mako

```diff
@@ -117,6 +117,12 @@
                                 <li><a href="${srRoot}/home/"><i class="menu-icon-home"></i>&nbsp;Show List</a></li>
                                 <li><a href="${srRoot}/home/addShows/"><i class="menu-icon-addshow"></i>&nbsp;Add Shows</a></li>
                                 <li><a href="${srRoot}/home/postprocess/"><i class="menu-icon-postprocess"></i>&nbsp;Manual Post-Processing</a></li>
+                                % if sickbeard.SHOWS_RECENT:
```

Comment:
> And I think that RECENT_SHOWS is a better name.

@MGaetan89 I named it SHOWS_ because I like to kind of "namespace" my variables. That way I can group all "shows" (or whatever) related variables together. If that isn't the desired or appropriate style, no problem.